templates/index.html

```diff
@@ -36,7 +36,8 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
                   </li>
                   <li>
                     <i class="fa fa-location-arrow"></i>
-                    <span>Delhi-NCR</span>
+                    <!-- Changing the venue from "Delhi, NCR" to "SSCBS" -->
+                    <span align="center"><a style="color:#FFF;" href="https://www.google.co.in/maps/place/Shaheed+Sukhdev+College+of+Business+Studies" target="_blank"> Shaheed Sukhdev College<br /> Of Bussiness Studies</a></span>                    
```

Comment:
You can simply use `<br>` instead of `<br />`.

Reply:
Ok, on it.

Reply:
Done @realslimshanky